python/pyarrow/pandas_compat.py

```diff
@@ -1009,12 +1009,15 @@ def _is_generated_index_name(name):
     return re.match(pattern, name) is not None
 
 
+# ARROW-9096: added integer and floating
```

Comment:
nit: generally only have comment like this for TODOs.  git blame/git log can track when things were added.